### ISSUE TITLE
Send Notification Multiple Times

### ISSUE THREAD

**Issue by Avakining** [antazoey/patchday-ios#79]
Would it be possible to make it so that the app will send the notification multiple times (preferably configurable), so that if the user misses the only notification they don't forget to take their HRT?

For instance, have the current system with notifying x minutes before, but if the patch isn't changed y minutes before, renotify, then z minutes after, renotify (etc). Obviously there has to be a limit to how many, but this is a solution to the problem I have of sometimes missing the notification and not applying a new patch until after I wake up the next day.

**Comment by antazoey** [comment]
yes i think that is a good idea. i can make it configurable with a reasonable default.
also sometimes when a patch expires overnight, it might make sense to re-notify in the morning some time (by default)
